app/src/main/AndroidManifest.xml

```diff
@@ -4,9 +4,8 @@
 
     <uses-permission android:name="android.permission.INTERNET"/>
 
-        <!--android:name="com.activeandroid.app.Application"-->
     <application
-        android:name="com.paypal.psix.PSixApplication"
```

Comment:
We need that class as the start in the manifest, since it needs to set the context for other classes to use.